content/de/how-it-works.md

```diff
@@ -8,57 +8,59 @@ lastmod: 2019-10-18
 
 {{< lastmod >}}
 
-Das Ziel von Let's Encrypt und des [ACME-Protokolls](https://tools.ietf.org/html/rfc8555) besteht darin, die Einrichtung eines HTTPS-Servers zu ermöglichen, der automatisch ein vertrauenswürdiges Browserzertifikat ohne menschliches Eingreifen erhält. Dies wird durch Ausführen eines Zertifikatsverwaltungsagenten auf dem Webserver erreicht.
+Das Ziel von Let's&nbsp;Encrypt und des [ACME-Protokolls](https://tools.ietf.org/html/rfc8555) besteht darin, die Einrichtung eines HTTPS-Servers zu ermöglichen, der automatisch ein vertrauenswürdiges Browserzertifikat ohne menschliches Eingreifen erhält.  Dies wird durch Ausführen eines Zertifikatsverwaltungsagenten auf dem Webserver erreicht.
```

Comment:
Das Ziel von Let's Encrypt und des [ACME-Protokolls](https://tools.ietf.org/html/rfc8555) besteht darin, die Einrichtung eines HTTPS-Servers zu ermöglichen, der automatisch ein vertrauenswürdiges Browserzertifikat ohne menschliches Eingreifen erhält. Dies wird durch Ausführen eines Zertifikatsverwaltungsagenten auf dem Webserver erreicht.

Reply:
This is the existing translation without a change.

Reply:
This is actually the same issue as below with the double spaces.